### ISSUE TITLE
Is the rendering process of TextureUV differentiable?

### ISSUE THREAD

**Issue by Qianhewu** [facebookresearch/pytorch3d#848]
I have read the provided tutorials and have understood that meshes with TextureVertex and TextureAtlas can be rendered by differentiable renderers. Both of them run well in my training pipeline. Yet I did not find any example codes for computing gradients of **TextureUV** and it seems the gradient of loss did not propagate back to the TextureUV. So I am wondering does PyTorch3D support differentiable rendering of a mesh with TextureUV as well?

**Comment by gkioxari** [comment]
`TexturesUV` is differentiable with respect to the map image but not with respect to the uv coordinates. You can easily check this yourself if you backprop through a textured rendering and check the gradients for the variables.

**Comment by Qianhewu** [comment]
Thanks for your reply! It works well.
